views/stats-proofofplay-page.twig

```diff
@@ -193,7 +193,31 @@
                             if (type != "display")
                                 return "";
 
-                            return moment().startOf("day").seconds(data).format("H:mm:ss")
+                            let durationData = moment.duration(data, "seconds");
```

Comment:
We need to think about how this gets translated into different languages (the works months, day, etc)